src/TConsole.cpp

```diff
@@ -1096,6 +1096,11 @@ void TConsole::scrollUp(int lines)
 
     if (lowerAppears) {
         QTimer::singleShot(0, this, [this]() {  mUpperPane->scrollUp(mLowerPane->getRowCount()); });
+        if (!mpHost->mTutorialForSplitscreenScrollbackAlreadyShown) {
+            QString infoMsg = tr("[INFO] Split-screen scrollback activated. Press CTRL-ENTER to cancel.");
```

Comment:
It works well, but doesn't get the same colouring as the over info messages. What if you try adding in the `-` and the right amount of spacing?

![image](https://user-images.githubusercontent.com/110988/235411401-bf28cb38-1aa8-47c0-bfa4-304bbdfc01e1.png)

Reply:
Yeah, like Vadim says, the colouring is triggered by a specific wording and spacing:
```suggestion
            QString infoMsg = tr("[ INFO ]  - Split-screen scrollback activated. Press CTRL-ENTER to cancel.");
```